### PR TITLE
Update web-api-functional-testing.md

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
Added phpunit_rest.xml Web API functional test.

## Purpose of this pull request

This pull request (PR) added command for the phpunit_rest.xml.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/get-started/web-api-functional-testing.html
https://devdocs.magento.com/guides/v2.4/get-started/web-api-functional-testing.html

### CODE REVIEW

File: src/guides/v2.3/get-started/web-api-functional-testing.md [hunk 1]
```diff
@@ -107,8 +107,16 @@ interface AdapterInterface
...
 
    ```bash
    vendor/bin/phpunit --config ./dev/tests/api-functional/phpunit_soap.xml
    ```
+   
+   or
+   
+   Run `phpunit` using the `/dev/tests/api-functional/phpunit_rest.xml` configuration file:
+   
+    ```bash
```
Comment (BarnyShergold) [inline src/guides/v2.3/get-started/web-api-functional-testing.md:120]:
```suggestion
   ```bash
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
You have some linting errors - trailing spaces.

**Comment by RakeshJesadiya** [comment]
@BarnyShergold Fixed the linting errors. Please check again. Thanks.

**Comment by BarnyShergold** [comment]
You are still getting linting errors -
src/guides/v2.3/get-started/web-api-functional-testing.md:114: MD009 Trailing spaces
src/guides/v2.3/get-started/web-api-functional-testing.md:116: MD009 Trailing spaces
src/guides/v2.3/get-started/web-api-functional-testing.md:118: MD009 Trailing spaces
src/guides/v2.4/get-started/web-api-functional-testing.md:114: MD009 Trailing spaces
src/guides/v2.4/get-started/web-api-functional-testing.md:116: MD009 Trailing spaces
src/guides/v2.4/get-started/web-api-functional-testing.md:118: MD009 Trailing spaces

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**